src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/config/RadicleSettingsView.java

```diff
@@ -329,13 +331,17 @@ private void initComponents() {
         });
     }
 
+    private boolean isCompatibleVersion(String v) {
+        var version = new DefaultArtifactVersion(v.replace("\n", "").trim());
```

Comment:
Is `DefaultArtifactVersion` handling errors on its own?
How does it behave if `v` is empty or invalid?